queries.py

```diff
@@ -61,3 +61,7 @@ def filter_nodes_in_bounty_programme(nodes):
         .join(
            IdentityRegistration, Node.node_key == IdentityRegistration.identity
         ).filter(IdentityRegistration.payout_eth_address != "")
+
+
+def filter_nodes_by_node_type(nodes, node_type):
+    return nodes.filter(getattr(Node, 'node_type') == node_type)
```

Comment:
Why do we need `getattr`? Isn't this the same?
```suggestion
    return nodes.filter(Node.node_type == node_type)
```

Reply:
I've no clue why we need it, but it refuses to work if I don't use it.

Reply:
Python ¯\_(ツ)_/¯